Project-EFT Tests/DBTests.cs

```diff
@@ -0,0 +1,178 @@
+        Problem problem;
+
+        [SetUp]
+        public void Setup()
+        {
+
+            userID = 80;
+            fakeUserID = 0;
+            adminID = 2;
+
+
+            sub1 = new AnswerSubmission("you've solved the first problem", new DateTime(2021, 4, 21, 17, 40, 23), userID, false, 1);
+            sub2 = new AnswerSubmission("Pdwk dqg FV duh frro!", new DateTime(2021, 4, 21, 17, 40, 32), userID, true, 2);
+
+            usersSubmissions = new List<AnswerSubmission>();
+            usersSubmissions.Add(sub1);
+            usersSubmissions.Add(sub2);
+
+            adminSubmissions = new List<Submission>();
+            adminSubmissions.Add(new Submission("Deleted account with username: new19", new DateTime(2021, 4, 21, 19, 52, 37), 2));
+
+            problem = new Problem(1, "A Simple Caesar", "Decrypt the following Caesar cipher: brx'yh vroyhg wkh iluvw sureohp!", "you've solved the first problem!", -1, -1, 1);
+
+
+            userEmail = "doNotDelete@doNotDelete.com";
+            fakeUserEmail = "whyWouldAnyoneMakeThis@anEmail.com";
+
+            username = "doNotDelete";
+            fakeUsername = "thisOneHasToBePastTheUserNameCharacterLimitsRightthisOneHasToBePastTheUserNameCharacterLimitsRightthisOneHasToBePastTheUserNameCharacterLimitsRightthisOneHasToBePastTheUserNameCharacterLimitsRightthisOneHasToBePastTheUserNameCharacterLimitsRight";
+
+            Dictionary<int, List<AnswerSubmission>> userSubMap = new Dictionary<int, List<AnswerSubmission>>();
+            userSubMap.Add(1, new List<AnswerSubmission>() { sub1 });
+            userSubMap.Add(2, new List<AnswerSubmission>() { sub2 });
+
+            user = new StandardUser(username, "49e5d4e3749b2f33b75dddc984878cdc", userEmail, -1, -1, 80, userSubMap);
+
+            admin = new Admin("doNotDeleteAdmin", "7c008e2d5c3a4e283dd97ad218534ee7", "doNotDeleteAdmin@doNotDelete.com", adminID);
+
+
+            DBConnector.InitForTests();
+
+        }
+
+        
+
+
+        //gets/checks
+        [Test]
+        public void testCheckIfUserSubmissionTableExists()
+        {
+            Assert.True(DBConnector.CheckIfUserSubmissionTableExists(userID));
+
+            Assert.False(DBConnector.CheckIfUserSubmissionTableExists(fakeUserID));
+        }
+
+        [Test]
+        public void testGetAnswerSubmissionsByID()
+        {
+            //gets the submission list for the test user
+            List<AnswerSubmission> testResults = DBConnector.GetAnswerSubmissionsByID(userID);
+
+            //iterates through the specific list and checks to see if they equal the expected values set in the setup function
+            int counter = 0;
+            foreach(AnswerSubmission sub in testResults)
+            {
+                Assert.True(sub.IsEqual(usersSubmissions[counter]));
+                counter++;
+
+                if (counter > usersSubmissions.Count)
+                {
+                    break;
+                }
+            }
```

Comment:
should there be a test to ensure the returned list has the expected count?

Reply:
Updated it now.